### PR TITLE
translations: Don't translate markup or force English grammar

### PR DESCRIPTION
**Pull request by maflcko**

### CODE REVIEW

File: src/qt/sendcoinsdialog.cpp [hunk 1]
```diff
@@ -312,8 +312,9 @@ void SendCoinsDialog::on_sendButton_clicked()
         if(u != model->getOptionsModel()->getDisplayUnit())
             alternativeUnits.append(BitcoinUnits::formatHtmlWithUnit(u, totalAmount));
     }
-    questionString.append(tr("Total Amount %1<span style='font-size:10pt;font-weight:normal;'><br />(=%2)</span>")
-        .arg(BitcoinUnits::formatHtmlWithUnit(model->getOptionsModel()->getDisplayUnit(), totalAmount))
+    questionString.append(tr("Total Amount %1")
```
Comment (laanwj) [inline src/qt/sendcoinsdialog.cpp]: Yes, this translation message was really ugly

**Comment by laanwj** [comment]
utACK

**Comment by fanquake** [comment]
utACK

File: src/init.cpp [hunk 2]
```diff
@@ -499,6 +499,7 @@ std::string HelpMessage(HelpMessageMode mode)
 
 std::string LicenseInfo()
 {
+    // todo: remove url's from translations on next change
```
Comment (laanwj) [inline src/init.cpp]: nit: URLs (no apostrophe)

Comment (MarcoFalke) [inline src/init.cpp]: @laanwj  :white_check_mark: fixed nit

**Pull request merged**